docs/resources/task.md

```diff
@@ -30,22 +39,24 @@ resource "iterative_task" "task" {
 ### Required
 
 - `cloud` - (Required) Cloud provider to run the task on; valid values are `aws`, `gcp`, `az` and `k8s`.
-- `script` - (Required) Script to run (relative to `workdir.input`); must begin with a valid [shebang](<https://en.wikipedia.org/wiki/Shebang_(Unix)>). Can use a string, including a [heredoc](https://www.terraform.io/docs/language/expressions/strings.html#heredoc-strings), or the contents of a file returned by the [`file`](https://www.terraform.io/docs/language/functions/file.html) function.
+- `script` - (Required) Script to run (relative to `storage.workdir`); must begin with a valid [shebang](<https://en.wikipedia.org/wiki/Shebang_(Unix)>). Can use a string, including a [heredoc](https://www.terraform.io/docs/language/expressions/strings.html#heredoc-strings), or the contents of a file returned by the [`file`](https://www.terraform.io/docs/language/functions/file.html) function.
 
 ### Optional
 
 - `name` - (Optional) Deterministic task name.
 - `region` - (Optional) [Cloud region/zone](#cloud-regions) to run the task on.
 - `machine` - (Optional) See [Machine Types](#machine-types) below.
```

Comment:
Why is **Generic Machine Types** under "Development" in the nav? (What does Development even refer to here?) Feels like machine types should be under the same section as the `iterative_task` res ref. somehow.

Reply:
Nav structure idea:

TPI
- GS
- Guides
  + Auth
  + Azure K8s
- Ref
  + Task res
  + Machine types

Reply:
Abbreviations aren't part of the proposal; I was just being lazy.

Reply:
> I find rather dissonant is moving the Machine Types page under the Ref

What does Development refer to now? Maybe I'm not getting the current intended struct.

Reply:
An XY solution would be disguising “Machine types“ as a guide (e.g. how to choose machine types) 🤔

Reply:
Yeah either put it in the guide or in the ref IMO 😬

Reply:
Yeah that's fine, call the section **Resources** (just also use that word in links if possible, instead of "reference")